pkg/api/devops/v1alpha3/pipeline_template.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2022 The KubeSphere Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
```

Comment:
why don't start from `v1alpha1`?

Reply:
Please note that this is not an API, which contains template structure in Pipeline. The reason I separate them into single file is to decouple them with existing pipeline_type.go.